src/App.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import Player from "./Player";
+import {db} from "./firebase";
+import 'moment-timezone';
+
+function App() {
+  const [players, setPlayers]=useState([]);
+  const currentDate=new Date().toLocaleDateString();
+
+  useEffect(()=>{
+    db.collection("nflplayers").orderBy("FFP", "desc").limit(4).onSnapshot(snapshot=>{
+      setPlayers(snapshot.docs.map(doc=>({
+        id:doc.id, 
+        player: doc.data()
+      })));
+    })
+  },[]);
+  
+  return (
+    <div className="app">
+      <div className="component6">
+      <h2 className="title">Today's Hot Picks</h2>
+      <p className="date"> ---------{currentDate}---------</p>
+        <div className="players"> 
+        {
+          players.map(({id, player})=>(
+            <Player key={id} name={player.name} image={player.image} position={player.position} FFP={player.FFP}/>
+          ))
+        }  
+        </div> 
+      </div>
+    </div>
+  );
```

Comment:
we should probably move all of this out into it's own component, so we can keep the App component to just things like providers and routing (setting up the react router, even with just one page [I imagine we'll have more pages in the future], should be a good way to do that)

Reply:
Resolved